scripts/apache_install.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# stop nginx
+service nginx stop
+
+# only install apache2 if not present yet
+if ! [ -f /etc/apache2/apache2.conf ]
+then
+	# update
+	apt-get update
+
+	# install apache
+	apt-get install -y apache2 libapache2-mod-php7.0 libapache2-mod-auth-mysql
```

Comment:
that would mean Apache gets installed by default in the vbox  
that's exactly what I wanted to avoid

this adds packages to the vbox that quite some users will not be using  
the purpose of my pull request is to make it easier for those who want to work with Apache to use it just by changing one line in the configuration...